tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -348,16 +348,34 @@ module('Acceptance | guid file | registration files', hooks => {
     });
 
     test('Error and canceling edit', async function(this: ThisTestContext, assert) {
-        await visit(`/--file/${this.file.id}`);
+        setupOnerror((e: any) => assert.ok(e, 'Error is handled'));
+
+        const node = server.create('node', {
+            id: 'mtadt',
+            currentUserPermissions: [Permission.Read, Permission.Write],
+        });
+        const projectFile = server.create('file', {
+            target: node,
+            name: 'Test File',
+        });
+        const url = `/--file/${projectFile.id}`;
+
+        server.namespace = '/v2';
+        server.patch('/custom_file_metadata_records/:id', () => ({
+            errors: [{
+                detail: 'Could not patch metadata',
+                source: { pointer: 'points to nowhere' },
+            }],
+        }), 400);
+
+        await visit(url);
 
         await click('[data-test-edit-metadata-button]');
         await fillIn('[data-test-title-field] > div > textarea', 'A New Title');
-        await server.get(`/--file/${this.file.id}`, {}, 500);
-        await server.get(`/--file/${this.file.id}`, {}, 200);
-        assert.dom('[data-test-cancel-metadata-button]').exists();
+        await click('[data-test-save-metadata-button]');
+        assert.dom('#toast-container', document as any)
+            .hasTextContaining(t('osf-components.file-metadata-manager.error-saving-metadata'));
         await click('[data-test-cancel-metadata-button]');
```

Comment:
After canceling, I think it would be worth a quick check to assert the title is unchanged.

Reply:
I had this in there, let me add it back.

Reply:
Ok, this is now passing with verification for the non-updated title field following cancellation.

Reply:
The bug fix encountered with cancel is now working.